resources/views/admin/page/index.blade.php

```diff
@@ -95,122 +95,34 @@
-                    <a class="play-icon popup-with-zoom-anim" href="#small-dialog"><img src="images/v1.jpg" title="allbum-name"></a>
-                    <a class="button play-icon popup-with-zoom-anim" href="#small-dialog">Listen now</a>
-                </div>
-                
-                <div id="small-dialog" class="mfp-hide">
-                    <iframe src="https://player.vimeo.com/video/12985622"></iframe>
-                </div>
-                <div class="col-md-3 content-grid">
-                    <a class="play-icon popup-with-zoom-anim" href="#small-dialog"><img src="images/v2.jpg" title="allbum-name"></a>
-                    <a class="button play-icon popup-with-zoom-anim" href="#small-dialog">Listen now</a>
-                </div>
-                <div class="col-md-3 content-grid">
-                    <a class="play-icon popup-with-zoom-anim" href="#small-dialog"><img src="images/v3.jpg" title="allbum-name"></a>
-                    <a class="button play-icon popup-with-zoom-anim" href="#small-dialog">Listen now</a>
-                </div>
-                <div class="col-md-3 content-grid last-grid">
-                    <a class="play-icon popup-with-zoom-anim" href="#small-dialog"><img src="images/v4.jpg" title="allbum-name"></a>
-                    <a class="button play-icon popup-with-zoom-anim" href="#small-dialog">Listen now</a>
-                </div>
-                <div class="col-md-3 content-grid">
-                    <a class="play-icon popup-with-zoom-anim" href="#small-dialog"><img src="images/v5.jpg" title="allbum-name"></a>
-                    <a class="button play-icon popup-with-zoom-anim" href="#small-dialog">Listen now</a>
-                </div>
-                <div id="small-dialog" class="mfp-hide">
-                    <iframe src="https://player.vimeo.com/video/12985622"></iframe>
-                </div>
-                <div class="col-md-3 content-grid">
-                    <a class="play-icon popup-with-zoom-anim" href="#small-dialog"><img src="images/v6.jpg" title="allbum-name"></a>
-                    <a class="button play-icon popup-with-zoom-anim" href="#small-dialog">Listen now</a>
-                </div>
-                <div class="col-md-3 content-grid">
-                    <a class="play-icon popup-with-zoom-anim" href="#small-dialog"><img src="images/v7.jpg" title="allbum-name"></a>
-                    <a class="button play-icon popup-with-zoom-anim" href="#small-dialog">Listen now</a>
-                </div>
-                <div class="col-md-3 content-grid last-grid">
-                    <a class="play-icon popup-with-zoom-anim" href="#small-dialog"><img src="images/v8.jpg" title="allbum-name"></a>
-                    <a class="button play-icon popup-with-zoom-anim" href="#small-dialog">Listen now</a>
+                    <a class="play-icon popup-with-zoom-anim" href="{{ route('single','$song->id')}} "><img src="images/v1.jpg" title="allbum-name"></a>
```

Comment:
```suggestion
                    <a class="play-icon popup-with-zoom-anim" href="{{ route('single', '$song->id')}} "><img src="images/v1.jpg" title="allbum-name"></a>
```
convention và sửa lại config file `images/v1.jpg`